smart_ats/jobs/api/views.py

```diff
@@ -0,0 +1,34 @@
+from rest_framework import viewsets
+from rest_framework.decorators import action
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+
+from smart_ats.jobs.models import Job
+
+from .permissions import IsJobCompanyAdminOrReadOnly
+from .serializers import JobSerializer, JobWriterSerializer
+
+
+class JobViewSet(viewsets.ModelViewSet):
+    queryset = Job.objects.all()
+    permission_classes = (IsAuthenticated, IsJobCompanyAdminOrReadOnly)
+
+    def get_serializer_class(self):
+        if self.action in ["list", "retrieve"]:
```

Comment:
if self.action in ["list", "retrieve"]:
            return JobSerializer
        return JobWriterSerializer